modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/persistence/impl/CPDefinitionLocalizationPersistenceImpl.java

```diff
@@ -1146,7 +1146,7 @@ public CPDefinitionLocalization updateImpl(
 					SanitizerUtil.sanitize(
 						companyId, groupId, userId,
 						CPDefinitionLocalization.class.getName(),
-						cpDefinitionLocalizationId, ContentTypes.TEXT_HTML,
+						cpDefinitionLocalizationId, ContentTypes.TEXT_PLAIN,
```

Comment:
does this work with the vulnerability reported here https://liferay.atlassian.net/browse/LPD-3300?